### PR TITLE
Stopwatch

### PR DESCRIPTION
**Pull request by alionazherdetska**
- [DEMO LINK](https://alionazherdetska.github.io/layout_stop-watch/)

- [x] Keyframes implemented using from/to + transform with rotate property
- [x] Stopwatch is centered and has correct arrows size
- [x] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1 +1,79 @@
...
+
+.watch {
+  box-sizing: border-box;
+  width: 80vmin;
+  height: 80vmin;
+  border-radius: 50%;
+  border: 3px dashed black;
+  position: relative;
+
+  &__center {
```
Comment (Viktor-Kost) [inline src/styles/main.scss:20]: add bigger z-index to a center than hands have
Screenshot 2024-01-08 at 15 38 05

File: src/styles/main.scss [hunk 2]
```diff
@@ -1 +1,79 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 10px;
+}
+
+
+.watch {
+  box-sizing: border-box;
```
Comment (Viktor-Kost) [inline src/styles/main.scss:13]: move to `*` selector

File: src/styles/main.scss [hunk 3]
```diff
@@ -1 +1,79 @@
...
+
+  &__hour-hand {
+    width: 9px;
+    height: 100%;
+    background-color: blue;
+    position: absolute;
+    left: 0;
+    right: 0;
+    margin: auto;
+    animation: rotate-hour-hand 55s linear infinite;
```
Comment (Viktor-Kost) [inline src/styles/main.scss:38]:
```suggestion
    animation: rotate-hour-hand 60s linear infinite;
```

File: src/styles/main.scss [hunk 4]
```diff
@@ -1 +1,79 @@
...
+    width: 3px;
+    height: 41vmin;
+    background-color: green;
+  }
+
+  @keyframes rotate-minute-hand {
+    to {
+      transform: rotate(360deg);
+    }
+  }
```
Comment (Viktor-Kost) [inline src/styles/main.scss:78]: both minute hand and hour one have the same animation. Reuse
```suggestion
```

File: src/styles/main.scss [hunk 5]
```diff
@@ -1 +1,79 @@
...
+    animation: rotate-minute-hand 5s linear infinite;
+  }
+
+  &__minute-hand::before {
+    content: '';
+    position: absolute;
+    width: 3px;
+    height: 41vmin;
+    background-color: green;
+  }
```
Comment (Viktor-Kost) [inline src/styles/main.scss:72]: Follow DRY (Don't Repeat Yourself ) principle. 
There are a lot of repeatable styles according to `watch__minute-hand` selector. So use `placeholders`/`mixins`, common selector with `modifier` (BEM) or list selectors with the same rules

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

File: src/styles/main.scss (hunk 1 above)
Comment (alionazherdetska) [reply src/styles/main.scss:20]: Adding Z-index to the center doesn't change it. I have tried putting z-index: 5 to the center and smaller indexes to hands, but it doesn't affect. Thus, I would be greatful for other suggestions.

File: src/styles/main.scss (hunk 3 above)
Comment (sTorba24) [inline src/styles/main.scss:38]: still not resolved
image

File: src/styles/main.scss (hunk 4 above)
Comment (sTorba24) [reply src/styles/main.scss:78]: still not resolved
image

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Almost good! Fix all previous comments

**Pull request closed without merging**